src/components/PropEditors.js

```diff
@@ -167,7 +167,7 @@ class NumberEditor extends React.Component {
   }
 
   componentWillReceiveProps(props) {
-    this.setState({ value: String(props.value) });
```

Comment:
As far as I can tell, if a number is passed in, it doesn't get converted to a string.